### DISCUSSION TITLE
More features for bottom bar

### DISCUSSION THREAD

**Discussion by TheMan133** [TT-ReBORN/Georgia-ReBORN#141] [Ideas]
Hello, I think it would be nice if clicking on a track name redirected you to the album, or clicking on an artist name redirected you to the artist's songs (referring to the bottom music bar).

**Comment by TT-ReBORN** [comment]
Hi @TheMan133, 

do you mean when clicking on the artist/track in the lower bar it should auto-scroll to the now playing track in the playlist?
Lower bar options are available when right clicking in the lower bar area and when right clicking in the seekbar.

-TT

**Reply by takaji10** [reply]
I've occasionally thought about this as well. This is how I'd imagine it:
1. Clicking on the artist name takes you to a playlist containing all tracks with that artist name included.
2. Clicking on the album takes you to a playlist containing just that album.

It's possible to also just have the current playlist scroll to the starting point, but sometimes an artist can be included on other albums than their own, so you end up being taken to a spot where there's only one song from that artist.

**Reply by TT-ReBORN** [reply]
So it has the same behavior like when you click on the playlist hyperlinks in the header?
I will think about it after the RC-2 release.

**Reply by takaji10** [reply]
Yes, exactly!

**Comment by silverwings15** [comment]
speaking of which, i don't remember if i asked this before so i'll ask again: the playback order button in the bottom bar currently has 3 options: standard, repeat track, and shuffle. is it possible to also add repeat playlist to it?

**Reply by TT-ReBORN** [reply]
I was too lazy to implement all 7 playback order settings ( top menu Playback > Order ) into the playback order button
due to setting up and picking appropriate icons for all 7 settings, and I also do not use the others. 
That is why I have chosen only the primary ones.
I will take a look again at the fontAwesome glyphs, maybe there are some that can be used for the others...

**Reply by TT-ReBORN** [reply]
@silverwings15, 

playback order "Repeat playlist" has been implemented as of now, found an appropriate icon for that mode.
I won't implement the other modes though...

**Reply by silverwings15** [reply]
thank you very much, i think that should be sufficient for the vast majority of users

**Reply by silverwings15** [reply]
another idea just came to mind: can you add a visual indicator somewhere to indicate that "Stop playback after current track" is active?

**Reply by TT-ReBORN** [reply]
There is no space for that, the only thing would be to add a tooltip on the playback button that shows 
"Stop playback after current track" is active...

**Reply by silverwings15** [reply]
implementation is wholly up to you!

**Reply by TT-ReBORN** [reply]
Added playback mode indicator as lower bar transport button tooltips
- Displays "Stop after current", "Playback follows cursor" and "Cursor follows playback" when main tooltips are enabled
  via top menu Options > Player controls > Tooltip > Show main tooltips

**Reply by Vanscov** [reply]
Hi everyone. I'm sorry for arriving a little late lol. I just found this skin last night and I think it is awesome. I wonder how I can make the transport buttons bar thinner?
<img width="1302" height="648" alt="barra" src="https://github.com/user-attachments/assets/bdec3dde-1a94-4df0-946a-ffd326e24068" />
